### PR TITLE
HEEDLS-442 Fix typos and change cancel to back link

### PR DESCRIPTION
**Pull request by livzorn**
### JIRA link
[_HEEDLS-442_](https://softwiretech.atlassian.net/browse/HEEDLS-444)

### Description
Just fixed a few typos in the view, and changed the cancel link to a back link to match the wireframe. 

### Screenshots
localhost_5001_TrackingSystem_CourseSetup_24286_Manage_AutoRefreshOptions

-----
### Developer checks
(Leave tasks unticked if they haven't been appropriate for your ticket.)

I have:
- [x] Run the formatter and made sure there are no IDE errors.
- [ ] Written tests for the changes (accessibility tests, unit tests for controller, data services, services, view models, etc)
- [ ] Manually tested my work with and without JavaScript. Full manual testing guidelines can be found here: https://softwiretech.atlassian.net/wiki/spaces/HEE/pages/6703648740/Testing
- [ ] Updated/added documentation in Swiki and/or Readme. Links (if any) are below:
  - [doc_1_here](link_1_here)
- [ ] Updated my Jira ticket with information about other parts of the system that were touched as part of the MR and have to be sanity tested to ensure nothing’s broken.
- [x] Scanned over my own MR to ensure everything is as expected.

### CODE REVIEW

**Review by DanBloxham-sw — APPROVED** [review]
Looks like this fixes everything.

File: DigitalLearningSolutions.Web/Views/TrackingSystem/CourseSetup/ManageCourse/EditAutoRefreshOptions.cshtml [hunk 1]
```diff
@@ -32,23 +32,23 @@
...
       <vc:single-checkbox asp-for="@nameof(Model.ApplyLpDefaultsToSelfEnrol)"
                           label="Apply to self-enrolments"
-                          hint-text="Apply auto-refresh settings when a user has self enrolled on a course (rather than being enrolled by an administrator)" />
+                          hint-text="Apply auto-refresh settings when a user has self-enrolled on a course (rather than being enrolled by an administrator)" />
 
       <button class="nhsuk-button" type="submit">Save</button>
     </form>
 
-    <vc:cancel-link asp-controller="ManageCourse" asp-action="Index" asp-all-route-data="@cancelLinkData" />
+    <vc:back-link asp-controller="ManageCourse" asp-action="EditLearningPathwayDefaults" asp-all-route-data="@backLinkData" link-text="Go back" />
```
Comment (SteveJacksonSoft) [inline DigitalLearningSolutions.Web/Views/TrackingSystem/CourseSetup/ManageCourse/EditAutoRefreshOptions.cshtml:52]: The cancel link is better actually - it's consistent with the other edit pages on Course Setup.

**Review by SteveJacksonSoft — CHANGES_REQUESTED** [review]

File: DigitalLearningSolutions.Web/Views/TrackingSystem/CourseSetup/ManageCourse/EditAutoRefreshOptions.cshtml (hunk 1 above)
Comment (livzorn) [reply DigitalLearningSolutions.Web/Views/TrackingSystem/CourseSetup/ManageCourse/EditAutoRefreshOptions.cshtml]: changed it back :)

Comment (SteveJacksonSoft) [reply DigitalLearningSolutions.Web/Views/TrackingSystem/CourseSetup/ManageCourse/EditAutoRefreshOptions.cshtml]: Sorry... I was wrong, back-link is what we want.

Comment (livzorn) [reply DigitalLearningSolutions.Web/Views/TrackingSystem/CourseSetup/ManageCourse/EditAutoRefreshOptions.cshtml:52]: removed the last commit so it's a back link again

**Review by SteveJacksonSoft — APPROVED** [review]
Nice, merge away.

**Pull request merged**